base/filesystem.jl

```diff
@@ -45,10 +45,6 @@ export File,
        JL_O_CREAT,
        JL_O_EXCL,
        JL_O_TRUNC,
-       JL_O_TEMPORARY,
-       JL_O_SHORT_LIVED,
-       JL_O_SEQUENTIAL,
-       JL_O_RANDOM,
```

Comment:
These are defined, just perhaps not on your system: https://github.com/JuliaLang/julia/blob/master/src/file_constants.h

Reply:
On second thought, we should perhaps switch to the correct names for them (e.g. http://docs.libuv.org/en/v1.x/fs.html#c.UV_FS_O_SEQUENTIAL)

Reply:
I'll leave the choice to you, and just revert this change (#43817)